### PR TITLE
fix(helm): Ensure global volumes are mounted in memcached exporter

### PR DESCRIPTION
**Pull request by bradleypettit**
#### What this PR does

Recently global volume support was added to the helm chart, however it wasn't added to the exporter container in memcached statefulsets. This PR adds the missing reference in the exporter container template ensuring that global volumes are mounted as expected.

#### Which issue(s) this PR fixes or relates to

N/A

#### Checklist

- [ ] Tests updated.
- [ ] Documentation added.
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
- [ ] [`about-versioning.md`](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md) updated with experimental features.

### CODE REVIEW

File: operations/helm/charts/mimir-distributed/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 5.4.0
+version: 5.4.1
```
Comment (dimitarvdimitrov) [inline operations/helm/charts/mimir-distributed/Chart.yaml:2]: normally we don't do patch releases for these fixes. There are [weekly versions released](https://artifacthub.io/packages/helm/grafana/mimir-distributed/5.4.0-weekly.298) every Monday (usually on time) which contain the changes on main. Will these weekly releases work for you?

**Review by dimitarvdimitrov — COMMENTED** [review]
Thanks for the fix! I left a comment about the patch version bump

File: operations/helm/charts/mimir-distributed/Chart.yaml (hunk 1 above)
Comment (bradleypettit) [reply operations/helm/charts/mimir-distributed/Chart.yaml:2]: > Normally contributors need not bump the version.

My mistake, totally missed that. Weekly is definitely fine, I'll revert that change shortly.

**Review by dimitarvdimitrov — APPROVED** [review]
LGTM, thank you!

**Pull request merged**